### PR TITLE
Feat/author body

### PR DESCRIPTION
**Pull request by Nightapes**

### CODE REVIEW

File: internal/changelog/changelog.go [hunk 1]
```diff
@@ -129,8 +157,17 @@ func (c *Changelog) GenerateChangelog(templateConfig shared.ChangelogTemplateCon
...
+
 	for _, commits := range analyzedCommits {
 		for _, commit := range commits {
+
+			_, ok := authors[commit.Commit.Author]
+			if !ok {
+				authors[commit.Commit.Author] = 0
+			}
+			authors[commit.Commit.Author] = authors[commit.Commit.Author] + 1
+
```
Comment (fwiedmann) [inline internal/changelog/changelog.go:170]: Why do you count the contributor/commit? If we do not care about it we could use an empty struct here

File: internal/changelog/changelog.go [hunk 2]
```diff
@@ -145,12 +182,20 @@ func (c *Changelog) GenerateChangelog(templateConfig shared.ChangelogTemplateCon
...
+		Commits:          commitsPerScope,
+		BreakingChanges:  commitsBreakingChange,
+		Backtick:         "`",
+		Order:            order,
+		ShowBodyAsHeader: c.config.Changelog.ShowBodyAsHeader,
+		HasURL:           templateConfig.CommitURL != "",
+		URL:              templateConfig.CommitURL,
+	}
+
+	authorsNames := make([]string, len(authors))
```
Comment (fwiedmann) [inline internal/changelog/changelog.go:194]: also specify the cap. of the array. If not specified the backing underlying array will have a capacity of 2x the given length I think 🤪

Comment (fwiedmann) [reply internal/changelog/changelog.go:194]: Nope im wrong, when omitted will be the the same as the given length.

File: internal/changelog/changelog.go (hunk 1 above)
Comment (Nightapes) [reply internal/changelog/changelog.go:170]: At the beginning i showed the amounts of commits per author. But removed it in the template. I will remove it here too

**Pull request merged**